prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -62,7 +62,7 @@ open class Receiver(
      * the timezone they operate under, and what we can use, if present, to convert date times in their data to if they
      * so request.
      */
-    val timeZone: USTimeZone? = null,
+    val timeZone: USTimeZone? = USTimeZone.EASTERN,
```

Comment:
I don't think we should be setting a default here.  Is this something that was already happening just elsewhere?

Reply:
will check